recipe/meta.yaml

```diff
@@ -1,29 +1,27 @@
 {% set name = "pymeep" %}
 {% set version = "1.30.1" %}
-{% set sha256 = "4089bdfbda5acb51d1bcc7864d580121ef2648592611a59bebce807de26900c1" %}
-{% set buildnumber = 1 %}
+{% set build = 2 %}
 # recipe-lint fails if mpi is undefined
 {% set mpi = mpi or "nompi" %}
 {% if mpi == "nompi" %}
 # prioritize nompi via build number
-{% set buildnumber = buildnumber + 100 %}
+{% set build = build + 100 %}
 {% set mpi_prefix = "nompi" %}
 {% else %}
 {% set mpi_prefix = "mpi_" + mpi %}
 {% endif %}
 
 package:
-  name: pymeep
+  name: pymeep-split
```

Comment:
aliasing the name causes some strange stuff to happen, often you will see "name"-split being used to avoid this.